sdks/java/io/elasticsearch/pom.xml

```diff
@@ -86,11 +81,6 @@
       <version>4.5.2</version>
     </dependency>
 
-    <dependency>
-      <groupId>joda-time</groupId>
-      <artifactId>joda-time</artifactId>
```

Comment:
Yes, it's possible that elasticsearch IO doesn't use joda-time `Duration` or `Instant`. In that case, it makes sense to remove the dependency.